promotion/controllers.go

```diff
@@ -34,6 +34,9 @@ func Router(service *Service) chi.Router {
 	r.Method("GET", "/", middleware.InstrumentHandler("GetAvailablePromotions", GetAvailablePromotions(service)))
 	r.Method("POST", "/{promotionId}", middleware.HTTPSignedOnly(service)(middleware.InstrumentHandler("ClaimPromotion", ClaimPromotion(service))))
 	r.Method("GET", "/{promotionId}/claims/{claimId}", middleware.InstrumentHandler("GetClaim", GetClaim(service)))
+
+	r.Method("POST", "/orders", middleware.InstrumentHandler("CreateOrder", CreateOrder(service)))
```

Comment:
these need to be part of their own router. currenly the url for this is `/v1/promotions/orders` due to the router prefix

Reply:
we should probably create this new router and all other changes in an `orders` module

Reply:
https://github.com/brave-intl/bat-go/pull/226/commits/01550a28a522daa5ed645d120b70aa4037eaa745